build/yamls/antrea-eks.yml

```diff
@@ -407,12 +407,16 @@ data:
     #   tls.crt: <TLS certificate>
     #   tls.key: <TLS private key>
     #selfSignedCert: true
+
+    # Enable Antrea NetworkPolicy CRDs. Setting it to true enables the controller
+    # watch for ClusterNetworkPolicy CRDs.
+    #enableSecurityCRDs: false
```

Comment:
do you want to mention here as well that this is a temporary parameter while the implementation is not complete? I think you did it somewhere else

Reply:
This was in the comments of previous PR but not in the code. I can update it here to mention that